assets/src/edit-story/components/panels/animation/panel.js

```diff
@@ -79,6 +74,18 @@ function AnimationPanel({
   }, [selectedElements, selectedElementAnimations]);
 
   const elAnimationId = updatedAnimations[0]?.id;
+
+  const handlePanelChange = useCallback(
+    (animation, submitArg = false) => {
+      if (shallowEqual(animation, updatedAnimations[0])) {
+        return;
```

Comment:
Some input fields fire a change event with a new value on blur because the user could have changed the number. This prevents a double play of animations if nothing changed.